test/unit/builtins/string.spec.ts

```diff
@@ -265,3 +265,21 @@ test.each([
         `;
     expect(util.transpileAndExecute(code)).toBe(6);
 });
+
+const trimTestCases = [
```

Comment:
I haven't checked the spec, but from a quick look at MDN page it also has to handle `\r\n\t\f\v\uFEFF\xA0`

Reply:
right, I'll add some more testcases